src/Relation/Unary.agda

```diff
@@ -162,6 +162,11 @@ IUniversal P = ∀ {x} → x ∈ P
 
 syntax IUniversal P = ∀[ P ]
 
+-- Stability - instances of P are stable wrt double negation
+
+Stable : Pred A ℓ → Set _
+Stable P = ∀ x → Nullary.Stable (P x)
```

Comment:
Missing CHANGELOG entry?

Reply:
Oops!

Reply:
No worries! That's why we have a review process!

I'm happy to stick with the definition. It's consistent with us lifting up `Decidable`/`Irrelevant` etc, to unary and binary relations.

Reply:
Would you be happy with the definition
```agda
Stable = Universal (Nullary.Stable)
```
or is your/srtdlib's preference to write out the quantifiers in full? It seems as though, as more and more of such definitions get added, that we strive for some uniformity via higher-order combinators (but I understand about not exploiting the provided `syntax` declarations for `Universal` and `IUniversal`)...